ci-operator/step-registry/ipi/conf/nutanix/multi-disks/ipi-conf-nutanix-multi-disks-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "ipi/conf/nutanix/multi-disks/ipi-conf-nutanix-multi-disks-ref.yaml",
```

Comment:
```suggestion
	"path": "ipi/conf/nutanix/compute/data-disks/ipi-conf-nutanix-comput-data-disks-ref.yaml",
```

Reply:
Got it, I'll also move gpu to it since it's also only support on compute